app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+
+  def new
+    
+    @user = User.new
+  end
+
+  def create
+    
+    @user = User.new(params.require(:user).permit!)
```

Comment:
I should use instance variable, because of saving fail will need to render :new again.